src/ducks/transactions/TransactionPageErrors/errors.js

```diff
@@ -0,0 +1,53 @@
+import sortBy from 'lodash/sortBy'
+import get from 'lodash/get'
+import keyBy from 'lodash/keyBy'
+import mapValues from 'lodash/mapValues'
+import { isErrored, isBankTrigger } from 'utils/triggers'
+
+const getCreatedByApp = acc => get(acc, 'cozyMetadata.createdByApp')
```

Comment:
This and `isActionableError ` could be in a cozy-client models imo, but I don't know if that's possible while keeping them overridable?

Reply:
I am not sure I understand your question ? Any package can be overrided in an override repository.

Reply:
As far as I remember, overriding a dependency was a bit trickier, but maybe I misremember. But yeah in that case I think cozy-client models would be a good place for these functions.

Reply:
Done in https://github.com/cozy/cozy-client/pull/738